src/plotSearch/helpers.ts

```diff
@@ -16,3 +26,128 @@ export const getPageForCurrentPlotSearch = (
       return null;
   }
 };
+
+export const parseCustomDetailedPlanToTargetPlan = (
+  customDetailedPlanId: number,
+  customDetailedPlan: CustomDetailedPlan
+): TargetPlan => {
+  return {
+    id: customDetailedPlanId,
+    identifier: customDetailedPlan.identifier,
+    area: customDetailedPlan.area,
+    detailed_plan_identifier: customDetailedPlan.detailed_plan,
+    detailed_plan_latest_processing_date:
+      customDetailedPlan.detailed_plan_latest_processing_date,
+    detailed_plan_latest_processing_date_note:
+      customDetailedPlan.detailed_plan_latest_processing_date_note,
+    plan_unit_type: customDetailedPlan.type,
+    plan_unit_state: customDetailedPlan.state,
+    plan_unit_intended_use: customDetailedPlan.intended_use
+      ? customDetailedPlan.intended_use.id
+      : undefined,
+    geometry: customDetailedPlan.geometry,
+    address: customDetailedPlan.address,
+    usage_distributions: customDetailedPlan.usage_distributions,
+    info_links: customDetailedPlan.info_links,
+    preconstruction_estimated_construction_readiness_moment:
+      customDetailedPlan.preconstruction_estimated_construction_readiness_moment,
+    rent_build_permission: customDetailedPlan.rent_build_permission,
```

Comment:
This doesn't currently show because the data was not yet available for regular plan units. The placeholder on L183 of mapSearchSingleTargetView.jsx should be removed, and  the favourite card component needs to likewise be adjusted.

Reply:
Sorry, there's one more place this needs to be added to that I spotted (the main list view of all plot searches has a column for this).